BedrockCore.cpp

```diff
@@ -369,19 +369,22 @@ void BedrockCore::postProcessCommand(unique_ptr<BedrockCommand>& command, bool i
 
 void BedrockCore::_handleCommandException(unique_ptr<BedrockCommand>& command, const SException& e, SQLite* db, const BedrockServer* server) {
     string msg = "Error processing command '" + command->request.methodLine + "' (" + e.what() + "), ignoring.";
+
+    STable logParams;
     if (!e.body.empty()) {
-        msg = msg + " Request body: " + e.body;
+        logParams = SParseJSONObject(e.body);
```

Comment:
Are we sure `e.body` is always JSON, @tylerkaraszewski ?

Reply:
I was looking at SParseJSONObject and it seems like it will just return empty if it's invalid JSON so that should be fine but I guess this will clear out any string bodies